src/components/desktop/Header.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components'
+
+// Might become a navbar, so use <nav> for this
+const DesktopHeader = styled.header`
```

Comment:
Are you suggesting you wanted to use styled.nav instead of styled.header?

Reply:
Yes, but not totally sure if this header is a nav yet. Waiting on
@SirajChokshi's content inside the bar.